randomfiletree/test/test_cli.py

```diff
@@ -11,7 +11,7 @@
 class TestCli(unittest.TestCase):
     def test_cli(self):
         with tempfile.TemporaryDirectory() as dirname:
-            subprocess.run(["python3", "randomfiletree", dirname])
+            subprocess.run(["randomfiletree", dirname], shell=True)
```

Comment:
Sorry, but why do we need `shell=True`? We're passing a list of arguments
![image](https://user-images.githubusercontent.com/13602468/193461017-ce9f118a-8d90-4093-9a4e-f7ea7577adc4.png)

Reply:
I added that, because i was getting this error right here when running `nose2` command locally.

![image](https://user-images.githubusercontent.com/61963664/193462204-1fcc65b2-1d41-4277-8408-040e3ba302fa.png)

The fix was mentioned in here:

https://stackoverflow.com/questions/73193119/python-filenotfounderror-winerror-2-the-system-cannot-find-the-file-specifie